packages/openneuro-components/src/form/RadioWrap.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+import './forms.scss'
+
+export interface RadioWrapProps {
+  children: object
+  radioWrapID: string
+  layout: string
+}
+
+export const RadioWrap: React.FC<RadioWrapProps> = ({
```

Comment:
So this is an idea on another way we can implement this. We could make this into a `RadioGroup` component that takes an array of `Radio` props (ie `[ { label: 'Apple' value: 'apple' } ]` and then have this component loop through it and make the radio buttons there. We can talk about more specifics if it's something you think we can do/explore.